BlazorTooling.sln

```diff
@@ -1,112 +0,0 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
The story here is that everything in one .sln is fine. The command line build will do the right thing now if you're non-windows.